.env.example

```diff
@@ -1 +1,2 @@
-PORT = 
\ No newline at end of file
+PORT = 3000
+URL  = `mongodb://localhost:27017/wave_coding`  
```

Comment:
@hsedaqat1378 please remove the values from `.env.example` just leave the env variable with `=`. like `PORT=`.

Reply:
that be easy to change